analysis_options.yaml

```diff
@@ -0,0 +1,34 @@
+analyzer:
+  strong-mode: true
+  errors:
+    unused_import: error
+    unused_local_variable: error
+    dead_code: error
+linter:
+  rules:
```

Comment:
These are the ones we use on angular – and google3.

They are used a lot internally. Pretty standard set.